tests/test_io_converters.py

```diff
@@ -0,0 +1,109 @@
+
+
+def test_merge_ctm_filename(output_folder):
+    run(f'meeteval-io ctm2stm {example_files / "hyp1.ctm"} {example_files / "hyp2.ctm"} {output_folder / "hyp.stm"}')
+    assert (output_folder / "hyp.stm").exists()
+    meeteval.io.load(output_folder / "hyp.stm").to_seglst().unique('speaker') == {'hyp1', 'hyp2'}
+
+
+def test_merge_ctm_speaker_arg(output_folder):
+    run(f'meeteval-io ctm2stm --speaker spk-A {example_files / "hyp1.ctm"} {example_files / "hyp2.ctm"} {output_folder / "hyp.stm"}')
+    assert (output_folder / "hyp.stm").exists()
+    meeteval.io.load(output_folder / "hyp.stm").to_seglst().unique('speaker') == {'spk-A'}
+
+def test_piping(output_folder):
+    run(f'cat {example_files / "hyp.stm"} | meeteval-io stm2rttm {output_folder / "hyp.rttm"}')
+    run(f'cat {example_files / "hyp.stm"} | meeteval-io stm2rttm -')
+    run(f'cat {example_files / "hyp.stm"} | meeteval-io stm2rttm - | echo')
+    run(f'cat {example_files / "hyp.stm"} | meeteval-io stm2rttm - > {output_folder / "hyp.rttm"}')
+    run(f'meeteval-io stm2rttm <(cat {example_files / "hyp.stm"}) -')
+
+def test_convert_correct(output_folder):
+    run(f'meeteval-io stm2rttm {example_files / "hyp.stm"} {output_folder / "hyp.rttm"}')
+    assert (output_folder / "hyp.rttm").read_text() == (example_files / "hyp.rttm").read_text()
+
+    # TODO: dump timestamps as str or int / float?
+    # run(f'meeteval-io stm2seglst {example_files / "hyp.stm"} {output_folder / "hyp.seglst.json"}')
+    # assert (output_folder / "hyp.seglst.json").read_text() == (example_files / "hyp.seglst.json").read_text()
+
+    run(f'meeteval-io rttm2stm {example_files / "hyp.rttm"} {output_folder / "hyp.stm"}')
+    assert meeteval.io.load(output_folder / "hyp.stm") == meeteval.io.load(example_files / "hyp.stm").map(lambda l: l.replace(transcript='<NA>'))
+
+    run(f'meeteval-io seglst2stm {example_files / "hyp.seglst.json"} {output_folder / "hyp.stm"} -f')
+    assert meeteval.io.load(output_folder / "hyp.stm") == meeteval.io.load(example_files / "hyp.stm")
+
+def test_convert_file_exists(output_folder):
+    run(f'meeteval-io stm2rttm {example_files / "hyp.stm"} {output_folder / "hyp.rttm"}')
+
+    with pytest.raises(Exception, match='.*Output file .* already exists.* Use --force / -f to overwrite.'):
+        run(f'meeteval-io stm2rttm {example_files / "hyp.stm"} {output_folder / "hyp.rttm"}')
+
+    run(f'meeteval-io stm2rttm --force {example_files / "hyp.stm"} {output_folder / "hyp.rttm"}')
+    run(f'meeteval-io stm2rttm -f {example_files / "hyp.stm"} {output_folder / "hyp.rttm"}')
+
+def test_ctm_piping():
+    run(f'cat {example_files / "hyp1.ctm"} | meeteval-io ctm2stm --speaker spk-A - > /dev/null')
+    with pytest.raises(Exception, match='.*the following arguments are required: --speaker.*'):
+        run(f'cat {example_files / "hyp1.ctm"} | meeteval-io ctm2stm - > /dev/null')
+    run(f'meeteval-io ctm2stm <(cat {example_files / "hyp1.ctm"}) - > /dev/null')
```

Comment:
Should this fail? The file will be something like `/dev/fd/63`, which can be confusing.

Reply:
Agreed, this is confusing.

Reply:
'{filestem}'  is now disallowed if the file is not a regular file